sdk/communication/azure-communication-phonenumbers/pom.xml

```diff
@@ -186,20 +186,12 @@
       <activation>
         <jdk>[9,)</jdk>
```

Comment:
It would be good to standardize this activation version to 11. We are using 11 in most places except 6 libraries.